.github/workflows/delete-model-runs.yaml

```diff
@@ -0,0 +1,58 @@
+# Workflow that can be manually dispatched to delete test model runs that
+# do not need to be persisted indefinitely.
+#
+# Gated such that it's impossible to delete runs older than the current
+# assessment cycle, where each assessment cycle starts in April.
+
+name: delete-model-runs
+
+on:
+  workflow_dispatch:
+    inputs:
+      run-ids:
+        description: >
+          Comma-delimited list of IDs of model runs to delete (no spaces). Note
+          that the workflow assumes these IDs correspond to model runs for the
+          current upcoming assessment cycle, and if that's not the case the
+          deletion script will raise an error.
+        required: true
+        type: string
+
+jobs:
+  delete-model-runs:
+    runs-on: ubuntu-latest
+    permissions:
+      # Needed to interact with GitHub's OIDC Token endpoint so we can auth AWS
+      contents: read
+      id-token: write
+    steps:
+      - name: Checkout repo code
+        uses: actions/checkout@v4
+
+      - name: Setup R
+        uses: r-lib/actions/setup-r@v2
+        with:
+          use-public-rspm: true
+
+      - name: Install system dependencies
+        run: sudo apt-get install libgit2-dev
+        shell: bash
```

Comment:
This dependency is not cached, but [it seems to take only 3-4s to complete](https://github.com/ccao-data/model-res-avm/actions/runs/6908697011/job/18798517688#step:4:1), and [caching apt packages on GitHub Actions is complicated](https://stackoverflow.com/a/59277514), so I decided to keep things simple and skip a cache.